overrides/default-settings.gschema.override

```diff
@@ -100,7 +100,7 @@ default-search-engine='Google'
 
 [org.gnome.Epiphany.ui]
 expand-tabs-bar=false
-tabs-bar-visibility-policy='always'
+tabs-bar-visibility-policy='no-third-party'
```

Comment:
Is this really the setting for cookies? There is a cookies setting a few lines below which is set to `always` at the moment.